<launch>
    <arg name="pub_lidar_tf"  default="true" />
    <arg name="omv_equivalent"  default="false" />
    <arg name="pub_odom"  default="true" />
    <node pkg="vehicle_kinematics" type="vehicle_kinematics_node" name="vehicle_kinematics_node" output="screen">
        <rosparam file="/opt/robot_data/vehicle_config/vehicle_mechanical_config.yaml" command="load"/>
        <rosparam file="/home/zc/.ros/vehicle_config/odom_calibrate_config.yaml" command="load"/>
        <param name="pub_lidar_tf"   value="$(arg pub_lidar_tf)" /> 
        <param name="pub_odom"   value="$(arg pub_odom)" /> 
        <param name="omv_equivalent"   value="$(arg omv_equivalent)" /> 
        <param name="vehicle_type" type="string" value="single_servo" />
        <param name="config_file" type="string" value="$(find common)/vehicle_config/" />

        <!-- <remap from="/servo_cmd" to="/servo_cmd_k" />  
        <remap from="/odom" to="/odom_k" />  
        <remap from="/auto_calibrate_odom" to="/auto_calibrate_odom_k" /> -->
    </node>
</launch>
